realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -200,6 +244,9 @@ public boolean contains(Object object) {
     public E get(int location) {
         E obj;
         realm.checkIfValid();
+        if (!isTablePresent()) {
```

Comment:
If this is to be removed, it goes to `BaseRealm.get()` and tries to get the removed `Table`. Is it fine?